src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrScanner.cs

```diff
@@ -62,6 +63,22 @@ await _dependencyTracker
     }
 
     protected abstract string ScanType { get; }
-    protected abstract Task<IEnumerable<string>> ScanRepository(SourceMapping sourceMapping, CancellationToken cancellationToken);
+    protected abstract Task<IEnumerable<string>> ScanRepository(SourceMapping sourceMapping, string baselineFilesPath, CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Returns a list of files that will be exluded from the scan operation, loaded from the baselineFilesPath
+    /// </summary>
+    /// <param name="baselineFilesPath"></param>
+    /// <returns></returns>
+    protected IEnumerable<string> GetBaselineFiles(string baselineFilesPath)
+    {
+        var a =  File.ReadAllLines(baselineFilesPath).Select(line =>
```

Comment:
Can you use `IFileSystem` to open the file? We can have unit test for this later easier that way.